BSP/STM32F413/Src/BSP_PWM.c

```diff
@@ -0,0 +1,170 @@
+ /* Copyright (c) 2018-2022 UT Longhorn Racing Solar */
+
+#include "BSP_PWM.h"
+
+GPIO_InitTypeDef GPIO_INIT_STRUCT; //struct used to initialize pins
```

Comment:
The old code (and other examples I saw online) used separate GPIO_InitTypeDef variables for each port so it's possible this *may* be causing our contactor control issue for the fan contactor but I doubt it